core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java

```diff
@@ -185,7 +186,11 @@ private void register() throws IOException {
             Configuration.getConfiguration(Scope.WORKER));
         LOG.info("Worker registered with worker ID: {}", mWorkerId.get());
         String dbDir = Configuration.getString(PropertyKey.DORA_WORKER_METASTORE_ROCKSDB_DIR);
-        mMetaStore = new RocksDBDoraMetaStore(dbDir + ".worker." + mWorkerId);
+        Duration duration =
+            Configuration.getDuration(PropertyKey.DORA_WORKER_METASTORE_ROCKSDB_TTL);
+        long ttl = duration.isZero() || duration.isNegative() ? -1 : duration.getSeconds();
+
+        mMetaStore = new RocksDBDoraMetaStore(dbDir + ".worker." + mWorkerId, ttl);
```

Comment:
Is the logic here consistent with https://github.com/Alluxio/alluxio/pull/16960/files#diff-9e60ad35cce49cbaad423bcf27ac539cc9f8ec4ef5bfc2fb0105b7a03e4f6c30

Reply:
After #16960 is merged, I will rebase this patch.

They are two independent patches for different purposes.

Reply:
This PR is rebased.
Please check again.